### PR TITLE
Various grammar changes

### PR DESCRIPTION
**Pull request by Mohit-Nain**
Since sounds more appropriate than while in line 129

### CODE REVIEW

**Comment by KristofferC** [comment]
Hm, I personally feel the "while" works well because it is sort of saying that even though there are no hard rules, there are still some conventions. But the conventions are not a direct consequence of the lack of rules... Maybe :)

File: doc/src/manual/integers-and-floating-point-numbers.md [hunk 1]
```diff
@@ -99,16 +99,16 @@ julia> UInt
...
+64 bits, always create 64-bit integers, regardless of the system type:
 
 ```jldoctest
 # 32-bit or 64-bit system:
 julia> typeof(3000000000)
 Int64
 ```
 
-Unsigned integers are input and output using the `0x` prefix and hexadecimal (base 16) digits
+Unsigned integers are used as input and output using the `0x` prefix and hexadecimal (base 16) digits
```
Comment (rfourquet) [inline doc/src/manual/integers-and-floating-point-numbers.md]: I was understanding `input` and `output` as verbs, so the initial version looks better to me. Moreover, in this new version, "used" and "using" are used too close in the same sentence.

Comment (mcabbott) [reply doc/src/manual/integers-and-floating-point-numbers.md]: You could say something like "are typed in and printed out using the"...

Comment (StefanKarpinski) [reply doc/src/manual/integers-and-floating-point-numbers.md]: "typed" is pretty overloaded in the programming context. I think the original is quite clear.

**Comment by KristofferC** [comment]
Has some issues that have been brought up in the comments. Feel free to open a new PR with these fixed.

**Pull request closed without merging**